virt/virt-2-4-release-notes.adoc

```diff
@@ -122,6 +122,13 @@ single sidebar menu item labeled *Virtualization*. When you click *Virtualizatio
 //https://bugzilla.redhat.com/show_bug.cig?=id=1959237
 * If you upgrade to {VirtProductName} {VirtVersion}, both older and newer versions of common templates are available for each combination of operating system, workload, and flavor. When you create a virtual machine by using a common template, you must use the newer version of the template. Disregard the older version to avoid issues. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1859237[*BZ#1859237*])
 
+* Running virtual machines that cannot be live migrated may block an OpenShift cluster upgrade. This includes virtual machines that use hostpath-provisioner storage or SRIOV network interfaces.
+As a workaround, reconfigure the virtual machines so that they can be powered off during a cluster upgrade. In the `spec` section of the virtual machine configuration file:
```

Comment:
Is the configuration in a file? Which API is this?

Reply:
This is the VirtualMachine object in the KubeVirt API